Fix EZP-24311: Clarify the differences between ezpublish-community and ezplatform repositories in README.md documentation

brookinsconsulting opened the pull request:
Hello,

Related issue for these pull requests: https://jira.ez.no/browse/EZP-24311

Per request from Andre to update the repository README.md documentation to clarify the differences between the older ezpublish-community repository and the new ezplatform repository in order to re-start community builds for the 5.x branch we are creating this issue.

Goal: Clarify the differences between the older ezpublish-community repository and the new ezplatform repository in repository README.md documentation.

To achieve this goal we will update the README.md documentation in each repository via separate pull requests to explain the main difference (today) between each repository is the inclusion or exclusion of legacy dependencies by default.


This pull request addresses the issue in question for this repository.
Another pull request exists for the other repository.
See issue ticket for alternate pull request links.

Please let us know how this finds you.

Thank you again for your continued support.

Cheers,
Brookins Consulting

sdaoudi commented on README.md:
```diff
@@ -39,6 +39,19 @@ features for editorial teams and media companies, 100% built on top of *eZ Platf
+This repository provides eZ Platform the Symfony based cms solution without all the eZ Publish Legacy dependancies by default, though these may be added manually.
+
+## Comparison
+* eZ Platform : eZ Publish Legacy dependancies not included by default, though they can be manually installed.
```
dependEncies and not dependAncies

sdaoudi commented on README.md:
```diff
@@ -39,6 +39,19 @@ features for editorial teams and media companies, 100% built on top of *eZ Platf
+## Comparison
+* eZ Platform : eZ Publish Legacy dependancies not included by default, though they can be manually installed.
+
+* eZ Publish 5 : eZ Publish Legacy dependancies are included by default.
```
dependEncies and not dependAncies

brookinsconsulting commented:
Hello @sdaoudi 

Thank you for your prompt response and feedback. We have addressed the issues you brought up and rebased.

We have also done the same in the corresponding ezpublish-community PR, https://github.com/ezsystems/ezpublish-community/pull/239

Cheers,
Brookins Consulting

sdaoudi commented:
Thanks @brookinsconsulting :+1:

andrerom commented:
review ping @bdunogier

bdunogier commented on README.md:
```diff
@@ -6,6 +6,11 @@
+## eZ Publish / eZ Platform
+This repository contains the eZ Platform (formerly eZ Publish 6) the next generation cms which uses the same Symfony kernel as eZ Publish 5, but does not include the legacy application, nor the dependent libraries. eZ Platform is where the new features are added.
+
+[eZ Publish 5](https://github.com/ezsystems/ezpublish-community) repository integrates eZ Publish Legacy and the dependent libraries and is mostly maintained for backwards compatibility.
```
"*The* eZ Publish 5 repository, mostly maintained for backwards compatibility, integrates eZ Publish Legacy..."

bdunogier commented:
Let me know what you think of my suggestions. One last round, and we can merge this, as far as i am concerned.

brookinsconsulting commented:
Hello @bdunogier 

We have completed the changes requested and flattened.

Cheers,
Brookins Consulting

andrerom commented:
+1

brookinsconsulting commented:
ping @bdunogier 

I noticed you closed the issue ticket, but you should also merge the ezplatform PR for the same issue ticket, https://github.com/ezsystems/ezplatform/pull/12

Cheers,
Brookins Consulting

brookinsconsulting commented:
@bdunogier 

Thank you :)

Cheers,
Brookins Consulting
